src/test/java/reactor/netty/http/client/HttpClientTest.java

```diff
@@ -2093,11 +2089,11 @@ public void testIssue988() {
 		ConnectionProvider provider = ConnectionProvider.create("testIssue988", 1);
 		HttpClient client =
 				createHttpClientForContextWithAddress(provider)
-				        .tcpConfiguration(tcpClient -> tcpClient.wiretap("testIssue988", LogLevel.INFO)
-				                                                .metrics(true));
+				        .wiretap("testIssue988", LogLevel.INFO)
+				        .metrics(true, s -> s);
```

Comment:
I wonder how often we expect users to change the representation of the URI in the metrics tags... if the answer is "not often", maybe it would be worth it to add an overload that doesn't take the `Function`?

Reply:
This is https://github.com/reactor/reactor-netty/pull/1066 the PR from 0.9.x
The method `metrics(boolean, Function)` is not related to this PR.
The PR has adaptation for the fix in 0.9.x as I rebase on top of the master branch and adapt all changed that are merged to master.

Reply:
thanks for the reminder 👍